src/utils/call_openai_summarise.py

```diff
@@ -0,0 +1,34 @@
+from openai import OpenAI
+
+
+def create_openai_summary(
+    system_prompt: str,
+    user_prompt: str,
+    context: str,
+    open_api_key: str,
+) -> dict:
+    client = OpenAI(api_key=open_api_key)
+
+    user_prompt = user_prompt.format(context)
+
+    try:
+        messages = [
+            {"role": "system", "content": system_prompt},
+            {"role": "user", "content": user_prompt},
+        ]
+        completion = client.chat.completions.create(
+            messages=messages,  # type: ignore
+            max_tokens=250,
+            temperature=0.2,
+            model="gpt-3.5-turbo-0125",
```

Comment:
If we decide not to do the different summarisation by use case options, whether we could have "apply filters and GPT3.5" and "apply filters and GPT4" buttons to see difference in quality of content summarisation. If we want to make it as good as possible for UR we could justify using GPT4 because we aren't calling it much